src/Tgstation.Server.Host/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -161,6 +162,13 @@ public static void UseServerBranding(this IApplicationBuilder applicationBuilder
 			applicationBuilder.Use((context, next) =>
 			{
 				context.Response.Headers.Add("X-Powered-By", assemblyInformationProvider.VersionPrefix);
+				if (context.Request.Method == HttpMethods.Put)
+				{
+					var logger = GetLogger(context);
+					context.Response.Headers.Add(HeaderNames.Warning, "299 - HttpMethod.Put is depreciated and will be replaced with HttpMethod.Post in next major release.");
```

Comment:
Remove the reference to HTTP 299 as that's not going to affect the status code at all and its deprecated anyway.

Also `HttpMethod` is an ASP NET core implementation detail

```suggestion
					context.Response.Headers.Add(HeaderNames.Warning, "HTTP PATCH/PUT/DELETE verbs are depreciated and will be replaced with POST with updated endpoints in an upcoming major REST API release.");
```